### PR TITLE
3 packages from 0xffea/ocaml-redis at 0.4

### PR DESCRIPTION
**Pull request by c-cube**
This pull-request concerns:
-`redis.0.4`: Redis client
-`redis-lwt.0.4`: Redis client (lwt interface)
-`redis-sync.0.4`: Redis client (blocking)

---
* Homepage: https://github.com/0xffea/ocaml-redis
* Source repo: git+https://github.com/0xffea/ocaml-redis.git
* Bug tracker: https://github.com/0xffea/ocaml-redis/issues

---
:camel: Pull-request generated by opam-publish v2.0.0

### CODE REVIEW

File: packages/redis-sync/redis-sync.0.4/opam [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+]
+homepage: "https://github.com/0xffea/ocaml-redis"
+bug-reports: "https://github.com/0xffea/ocaml-redis/issues"
+license: "BSD3"
+tags: ["redis" "unix"]
+synopsis: "Redis client (blocking)"
+dev-repo: "git+https://github.com/0xffea/ocaml-redis.git"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
```
Comment (kit-ty-kate) [inline packages/redis-sync/redis-sync.0.4/opam]: I think the tests section should be removed in opam as the sandbox prevents any use of the network

File: packages/redis-lwt/redis-lwt.0.4/opam [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+]
+homepage: "https://github.com/0xffea/ocaml-redis"
+bug-reports: "https://github.com/0xffea/ocaml-redis/issues"
+license: "BSD3"
+tags: ["redis" "lwt"]
+dev-repo: "git+https://github.com/0xffea/ocaml-redis.git"
+synopsis: "Redis client (lwt interface)"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
```
Comment (kit-ty-kate) [inline packages/redis-lwt/redis-lwt.0.4/opam]: same here I think. Is that ok?

Comment (c-cube) [reply packages/redis-lwt/redis-lwt.0.4/opam]: I guess it's ok, yes, if there's no possibility of `opam install` with tests.

**Comment by kit-ty-kate** [comment]
All green now. Thanks! It might be worth returning the fixes upstream or using a slightly different sets of tests for opam so that we don't get to do the same thing next time.

Thanks a lot!

**Pull request merged**